af/af00213/src/test/java/se/inera/intyg/common/af00213/v1/model/converter/CertificateToInternalTest.java

```diff
@@ -48,6 +56,9 @@ void setup() {
                 .setId("id")
                 .setTextVersion("TextVersion")
                 .build();
+
+            texts = Mockito.mock(TreeMap.class);
```

Comment:
Can you see if you can mock this on "class"-level so we don't have to repeat it for each Nested test?